.github/workflows/chatops.yaml

```diff
@@ -33,9 +33,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Set up Python
-        uses: actions/setup-python@v1
+        uses: actions/setup-python@v5
         with:
-          python-version: 3.6
+          python-version: 3.11
```

Comment:
Always use quotes around strings like this, as yaml interprets it as a number, so 3.10 would actually be 3.1.